docs/Meadow/Meadow.OS/Cellular/index.md

```diff
@@ -4,116 +4,161 @@ title: Cellular
+**Cat-M1**, **NB-IoT**, and **GSM** (Global System for Mobile Communications) are all cellular network technologies with distinct characteristics. **GSM**, the most widely used cellular standard globally, provides higher data rates compared to **NB-IoT** and **Cat-M1** but has higher power consumption. In contrast, **Cat-M1** and **NB-IoT** are optimized for IoT applications. **Cat-M1** offers higher data rates and mobility, while **NB-IoT** provides ultra-low power consumption and extended coverage. The choice among **GSM**, **Cat-M1**, and **NB-IoT** depends on the specific requirements of the application, considering factors such as data rates, power consumption, and coverage area.
 
-| Modules / Network Modes   | Cat-M1 (LTE-M or eMTC) | NB-IoT | GSM/GRPS 2G |
-|--------------------------|------------------------|--------|-------------|
-| Quectel BG770A Cell Wing | ✔                  | ✔  | -        |
-| Quectel BG95-M3          | ✔                  | ✔  | ✔       |
-| Quectel M95              | -                   | -   | ✔       |
+# Supported Cellular Modules
 
+The cellular network for the Meadow platform is compatible with various modules, each supporting different network operation modes. The following table describes the network operation modes supported by each module:
 
-**Cat-M1**, **NB-IoT**, and **GSM** (Global System for Mobile Communications) are all cellular network technologies with distinct characteristics. **GSM**, the most widely used cellular standard globally, provides higher data rates compared to **NB-IoT** and **Cat-M1** but has higher power consumption. In contrast, **Cat-M1** and **NB-IoT** are optimized for IoT applications. **Cat-M1** offers higher data rates and mobility, while **NB-IoT** provides ultra-low power consumption and extended coverage. The choice among **GSM**, **Cat-M1**, and **NB-IoT** depends on the specific requirements of the application, considering factors such as data rates, power consumption, and coverage area.
+| Modules / Network Modes   | Cat-M1 (LTE-M or eMTC) | NB-IoT | GSM/GRPS 2G | GNSS   |
+|---------------------------|------------------------|--------|-------------|--------|
+| Quectel BG95-M3           | ✔                     | ✔      | ✔           | ✔      |
+| Quectel M95               | -                      | -      | ✔           | -      |
 
+# Hardware configuration
 
-## Network configuration
+## Quectel BG95M3 with NimbeLink Skywire click board
 
-Using Cellular Meadow, you don't need to understand the complexities of AT commands or invest time studying module datasheets to enable the cellular network interface.
+To setup the hardware, you could use a Skywire click adapter, which hosts NimbeLink/Skywire ™ cellular modems (using stacking headers) to MikroElektronika development boards. 
```

Comment:
Not a concern, but don't worry about the TM characters like this in the future. Sometimes, they can cause rendering issues in the pipeline.